docs/src/main/asciidoc/rest-data-panache.adoc

```diff
@@ -132,10 +134,10 @@ It can be used in your resource interface:
 @ResourceProperties(hal = true, path = "my-people")
 public interface PeopleResource extends PanacheEntityResource<Person, Long> {
     @MethodProperties(path = "all")
-    Response list();
+    Person list(Page page, Sort sort);
```

Comment:
Isn't it `List<Person>`?

Also it feels weird that these are `Page` and `Sort` when the generated code you showed up there uses `int` and `String`, is it normal?

Reply:
Yes, it's `List<Person>` my bad. I think I'll rewrite the doc to clarify the differences better.